lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -162,9 +164,55 @@ define((require, exports, module) ->
     ch = ContextHandler = {
       contextPreviewExists: false
       UPDATE_DELAY: 1000
+      LABEL_SEQUENCE: [
+        {type: "keyword", value: /^\\label$/}
+        {type: "lparen", value: /^\{$/}
+        {type: "variable.parameter", value: /.*/}
```

Comment:
It doesn't look like macros are even supported inside labels. It seems as though Katex, when parsing arguments, just searches for the first closing brace `}` and considers everything up to this as arguments. Which is actually different from main Papeeria compiler, where things like this:

    \label{abra{cadabra}}

are actually allowed and can be referenced with `\ref`. But even in main compiler, things like this

    \label{\begin{equation}\end{equation}}

result in an error. So, answering your question -- it **does** seem to always be a variable.parameter.